packages/frontend/package.json

```diff
@@ -26,7 +28,8 @@
     "start": "react-scripts start",
     "build": "react-scripts build",
     "test": "react-scripts test",
-    "eject": "react-scripts eject"
+    "eject": "react-scripts eject",
+    "generate": "npx openapi-typescript http://localhost:4200/api-json --output ./src/typing.ts"
```

Comment:
@GlockTian should we put typing.ts inside a folder that is dedicated for interfaces. i.e ./src/types/typing.ts Just in case if we have more interfaces later in the project.

Reply:
also we should probably name it something a bit more descriptive. Maybe `api-schema.ts`?
I would also like to change the command from `generate` to something more descriptive

Reply:
maybe `generate-api-types`. bit wordy though

Reply:
generate:api

Reply:
what about `generate-types:api` I feel like just generate isn't descriptive enough

Reply:
makes it sound as if the api is generated from here

Reply:
generate:api-types

Idea being if we ever add other generators then the script can just be called with something like `run-p generate:*` to run all scripts in the glob